symphony-bdk-core/build.gradle

```diff
@@ -37,7 +37,7 @@ dependencies {
     implementation 'org.mapstruct:mapstruct:1.3.1.Final'
     annotationProcessor 'org.mapstruct:mapstruct-processor:1.3.1.Final'
 
-    implementation 'org.apiguardian:apiguardian-api'
+    compile 'org.apiguardian:apiguardian-api'
```

Comment:
https://docs.gradle.org/current/userguide/java_library_plugin.html 

_The compile configuration still exists but should not be used as it will not offer the guarantees that the api and implementation configurations provide._

Did you have errors with the implementation scope?

Reply:
when I built the project, there are some warning about the enum `API.Status`. I had a look at the api-guardian [Readme](https://github.com/apiguardian-team/apiguardian), and changed to compile scope and the warning disappear.

Reply:
ah then we can add the java-library plugin in bdk-core and change it to 

    api 'org.apiguardian:apiguardian-api'
and we can do the same for jackson there are warnings for that too